### PR TITLE
🌐 [i18n-KO] Translated `model_doc/informer.md` to Korean

### PR DESCRIPTION
**Pull request by fabxoe**
# What does this PR do?

Translated the `model_doc/informer.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

@chhaewxn, @ahnjj, @jun048098, @fabxoe, @nuatmochoi, @heuristicwave

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@stevhliu May you please review this PR?

### CODE REVIEW

File: docs/source/ko/model_doc/informer.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+# Informer[[informer]]
+
+## Overview[[overview]]
+
+The Informer 모델은 Haoyi Zhou, Shanghang Zhang, Jieqi Peng, Shuai Zhang, Jianxin Li, Hui Xiong, Wancai Zhang가 제안한 [Informer: 장기 시퀀스 시계열 예측(LSTF)을 위한 더욱 효율적인 트랜스포머(Beyond Efficient Transformer)](https://arxiv.org/abs/2012.07436)라는 논문에서 소개되었습니다.
+
+이 방법은 확률적 어텐션 메커니즘을 도입하여 "게으른" 쿼리가 아닌 "활성" 쿼리를 선택하고, 희소 트랜스포머를 제공하여 기존 어텐션의 이차적 계산 및 메모리 요구사항을 완화합니다.
+
+해당 논문의 초록입니다:
```
Comment (ahnjj) [inline docs/source/ko/model_doc/informer.md:25]:
```suggestion
논문 초록:
```

File: docs/source/ko/model_doc/informer.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+*실제로 많은 응용프로그램에서는 장기 시퀀스 시계열 예측(LSTF)을 필요로 합니다. LSTF는 모델의 높은 예측 능력을 요구하며, 이는 출력과 입력간의 정확한 장거리 의존성 결합을 효율적으로 포착하는 능력입니다. 최근 연구에 따르면 트랜스포머가 예측 능력을 향상시킬 수 있는 잠재력을 보여주고 있습니다. 그러나 트랜스포머를 LSTF에 직접 적용하는 것을 막는 몇 가지 심각한 문제가 있습니다. 문제점으로는 이차 시간 복잡도, 높은 메모리 사용량, 인코더-디코더 아키텍처의 본질적 한계를 예로 들 수 있습니다. 이러한 문제를 해결하기 위해 우리는 LSTF를 위한 효율적인 트랜스포머 기반 모델인 Informer를 설계했습니다.
```
Comment (ahnjj) [inline docs/source/ko/model_doc/informer.md:27]:
```suggestion
*실제로 많은 응용프로그램에서는 장기 시퀀스 시계열 예측(LSTF)을 필요로 합니다. LSTF는 출력 - 입력 간 정확한 장기 의존성 결합도를 포착해내는 높은 예측 능력을 모델에 요구합니다. 최근 연구들은 예측 능력을 향상시킬 수 있는 트랜스포머의 잠재력을 보여주고 있습니다. 그러나, 트랜스포머를 LSTF에 직접 적용하지 못하도록 막는 몇 심각한 문제점들이 있습니다. 예로, 이차 시간 복잡도, 높은 메모리 사용량, 인코더-디코더 아키텍처의 본질적 한계를 들 수 있습니다. 이러한 문제를 해결하기 위해 LSTF를 위한 효율적인 트랜스포머 기반 모델인 Informer를 설계했습니다.
```

File: docs/source/ko/model_doc/informer.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+Informer의 세가지 독특한 특성:
+(i) ProbSparse 셀프 어텐션 메커니즘으로, 시간 복잡도와 메모리 사용량에서 O(L logL)를 달성하며 시퀀스 의존성 정렬에서 비교 가능한 성능을 보입니다. 
+(ii) 자기 주의 증류는 계단식 레이어 입력을 반으로 줄여 지배적인 주의를 강조하고 극도로 긴 입력 시퀀스를 효율적으로 처리합니다.
```
Comment (ahnjj) [inline docs/source/ko/model_doc/informer.md:31]:
```suggestion
(ii) 셀프 어텐션 증류는 계단식 레이어 입력을 반으로 줄여 지배적인 어텐션을 강조하고 극단적으로 긴 입력 시퀀스를 효율적으로 처리합니다. 
```

File: docs/source/ko/model_doc/informer.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+이 방법은 확률적 어텐션 메커니즘을 도입하여 "게으른" 쿼리가 아닌 "활성" 쿼리를 선택하고, 희소 트랜스포머를 제공하여 기존 어텐션의 이차적 계산 및 메모리 요구사항을 완화합니다.
+
+해당 논문의 초록입니다:
+
+*실제로 많은 응용프로그램에서는 장기 시퀀스 시계열 예측(LSTF)을 필요로 합니다. LSTF는 모델의 높은 예측 능력을 요구하며, 이는 출력과 입력간의 정확한 장거리 의존성 결합을 효율적으로 포착하는 능력입니다. 최근 연구에 따르면 트랜스포머가 예측 능력을 향상시킬 수 있는 잠재력을 보여주고 있습니다. 그러나 트랜스포머를 LSTF에 직접 적용하는 것을 막는 몇 가지 심각한 문제가 있습니다. 문제점으로는 이차 시간 복잡도, 높은 메모리 사용량, 인코더-디코더 아키텍처의 본질적 한계를 예로 들 수 있습니다. 이러한 문제를 해결하기 위해 우리는 LSTF를 위한 효율적인 트랜스포머 기반 모델인 Informer를 설계했습니다.
+
+Informer의 세가지 독특한 특성:
+(i) ProbSparse 셀프 어텐션 메커니즘으로, 시간 복잡도와 메모리 사용량에서 O(L logL)를 달성하며 시퀀스 의존성 정렬에서 비교 가능한 성능을 보입니다. 
+(ii) 자기 주의 증류는 계단식 레이어 입력을 반으로 줄여 지배적인 주의를 강조하고 극도로 긴 입력 시퀀스를 효율적으로 처리합니다. 
+(iii) 생성 스타일 디코더는 개념적으로 단순하지만 장기 시계열 시퀀스를 단계별 방식이 아닌 한 번의 전방 연산으로 예측하여 장기 시퀀스 예측의 추론 속도를 크게 향상시킵니다. 네 개의 대규모 데이터셋에 대한 광범위한 실험은 Informer가 기존 방법들을 크게 능가하며 LSTF 문제에 새로운 해결책을 제공함을 보여줍니다.*
```
Comment (ahnjj) [inline docs/source/ko/model_doc/informer.md:32]:
```suggestion
(iii) 생성 스타일 디코더는 개념적으로 단순하지만 장기 시계열 시퀀스를 단계별 방식이 아닌 한 번의 전방 연산으로 예측하여 장기 시퀀스 예측의 추론 속도를 크게 향상시킵니다. 4개의 대규모 데이터셋에 걸친 광범위한 실험은 Informer가 기존 방법들을 크게 능가하며 LSTF 문제에 새로운 해결책을 제공함을 보여줍니다.*
```

File: docs/source/ko/model_doc/informer.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+이 모델은 [elisim](https://huggingface.co/elisim)와 [kashif](https://huggingface.co/kashif)에 의해 기여 되었습니다.
+원본 코드는 [이곳](https://github.com/zhouhaoyi/Informer2020)에서 확인할 수 있습니다.
+
+## Resources[[resources]]
+
+시작하는 데 도움이 되는 Hugging Face와 community 자료 목록(🌎로 표시됨) 입니다. 여기에 포함될 자료를 제출하고 싶으시다면 PR(Pull Request)를 열어주세요. 리뷰 해드리겠습니다! 자료는 기존 자료를 복제하는 대신 새로운 내용을 담고 있어야 합니다.
```
Comment (ahnjj) [inline docs/source/ko/model_doc/informer.md:39]:
```suggestion
초심자분들께 도움이 될 Hugging Face와 community 공식 자료 목록(🌎로 표시됨) 입니다. 이 커뮤니티에 속할 자료를 제출하고 싶으시다면, PR(Pull Request)를 열어주세요. 리뷰 해드리겠습니다! 자료는 기존 자료를 복제하는 대신 새로운 내용을 담고 있어야 합니다.
```

**Review by ahnjj — CHANGES_REQUESTED** [review]

File: docs/source/ko/model_doc/informer.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+해당 논문의 초록입니다:
+
+*실제로 많은 응용프로그램에서는 장기 시퀀스 시계열 예측(LSTF)을 필요로 합니다. LSTF는 모델의 높은 예측 능력을 요구하며, 이는 출력과 입력간의 정확한 장거리 의존성 결합을 효율적으로 포착하는 능력입니다. 최근 연구에 따르면 트랜스포머가 예측 능력을 향상시킬 수 있는 잠재력을 보여주고 있습니다. 그러나 트랜스포머를 LSTF에 직접 적용하는 것을 막는 몇 가지 심각한 문제가 있습니다. 문제점으로는 이차 시간 복잡도, 높은 메모리 사용량, 인코더-디코더 아키텍처의 본질적 한계를 예로 들 수 있습니다. 이러한 문제를 해결하기 위해 우리는 LSTF를 위한 효율적인 트랜스포머 기반 모델인 Informer를 설계했습니다.
+
+Informer의 세가지 독특한 특성:
+(i) ProbSparse 셀프 어텐션 메커니즘으로, 시간 복잡도와 메모리 사용량에서 O(L logL)를 달성하며 시퀀스 의존성 정렬에서 비교 가능한 성능을 보입니다. 
+(ii) 자기 주의 증류는 계단식 레이어 입력을 반으로 줄여 지배적인 주의를 강조하고 극도로 긴 입력 시퀀스를 효율적으로 처리합니다. 
+(iii) 생성 스타일 디코더는 개념적으로 단순하지만 장기 시계열 시퀀스를 단계별 방식이 아닌 한 번의 전방 연산으로 예측하여 장기 시퀀스 예측의 추론 속도를 크게 향상시킵니다. 네 개의 대규모 데이터셋에 대한 광범위한 실험은 Informer가 기존 방법들을 크게 능가하며 LSTF 문제에 새로운 해결책을 제공함을 보여줍니다.*
+
+이 모델은 [elisim](https://huggingface.co/elisim)와 [kashif](https://huggingface.co/kashif)에 의해 기여 되었습니다.
```
Comment (jun048098) [inline docs/source/ko/model_doc/informer.md:34]:
```suggestion
이 모델은 [elisim](https://huggingface.co/elisim)와 [kashif](https://huggingface.co/kashif)가 기여했습니다.
```

**Review by jun048098 — COMMENTED** [review]
fix: docs suggestions

**Review by fabxoe — COMMENTED** [review]
fix: resolve suggestions

**Review by stevhliu — APPROVED** [review]
Thank you, just need to resolve the merge conflict!

**Pull request merged**